src/SubmitNotebook.ts

```diff
@@ -296,7 +296,7 @@ export class SubmitNotebook extends Widget implements Dialog.IBodyWidget<ISubmit
     let td_colspan4 = '<td style="padding: 1px;" colspan=4>';
     let subtitle = '<div style="font-size: var(--jp-ui-font-size3)">Environmental Variables</div>'
 
-    let html = '' + tr + td_colspan4 + subtitle + '</td>' + '</tr>';
+    let html = '';
```

Comment:
Should we just surround all these code in a `if (this._envVars.length > 0)`

Reply:
That works! I'll do that.